Playmate/View Controllers/Home Tab/HomeViewController.m

```diff
@@ -14,7 +14,7 @@
 #import "CalendarViewController.h"
 #import "ProfileViewController.h"
 
-@interface HomeViewController () <UITableViewDelegate, UITableViewDataSource, DZNEmptyDataSetSource, DZNEmptyDataSetDelegate>
```

Comment:
How come the table view is still working? 

Did you had the protocols also in the .h?

Reply:
Oops, this was a deletion in progress, the table view was not working at the time and I have since completely chaged the layout and implemented a collection view. no worries on this